site/docs/api/table-options.md

```diff
@@ -367,6 +367,20 @@ The table options are defined in `jQuery.fn.bootstrapTable.defaults`.
 
 - **Example:** [Detail View Icon](https://examples.bootstrap-table.com/#options/detail-view-icon.html)
 
+## detailViewHAlign
```

Comment:
I think we can use `detailViewAlign` instead.

Reply:
we already use "HAlign" for other options, I think it more constant or not ?

Reply:
There are options like this: `toolbarAlign`, `buttonsAlign`, `searchAlign`, `paginationHAlign`, `paginationVAlign`.

Reply:
Okay, will update it :)